cmd/internal/register/maps.go

```diff
@@ -11,7 +13,14 @@ import (
 
 func webMercatorMapFromConfigMap(cfg config.Map) (newMap atlas.Map) {
 	newMap = atlas.NewWebMercatorMap(string(cfg.Name))
-	newMap.Attribution = html.EscapeString(string(cfg.Attribution))
+
+	attribution := html.EscapeString(string(cfg.Attribution))
+	re := regexp.MustCompile(`&lt;(a\s(.+?)|/a)&gt;`)
+	tags := re.FindAllString(attribution, -1)
+	for _, tag := range tags {
+		attribution = strings.Replace(attribution, tag, html.UnescapeString(tag), 1)
+	}
```

Comment:
You could move this to a separate function and maybe call it `sanitizeAttribution`. That would allow you to add your test cases to the codebase which would be ideal.